code.google.com no longer supported

naoki-k opened the pull request:
Hello,
I came across the following error:

```
 $ go get code.google.com/p/google-api-go-client
package code.google.com/p/google-api-go-client: unable to detect version control system for code.google.com/ path
```

According to the below link, this is happening because Google Code completely shut down as of January 25th.
Thus, I fixed path from `code.google.com` to `google.golang.org`

reference: 
- http://google-opensource.blogspot.jp/2015/03/farewell-to-google-code.html
- https://code.google.com/archive/p/google-api-go-client/

broady commented on go/README:
```diff
@@ -1,15 +1,15 @@
 
-   go get code.google.com/p/goauth2/oauth
-   go get code.google.com/p/google-api-go-client/youtube/v3
+   go get google.golang.org/api/oauth2/v2
```
golang.org/x/oauth2

naoki-k replied:
Thank you for your comment.
I made a revision. Would you be able to check it?

odeke-em commented on go/oauth.go:
```diff
@@ -13,7 +13,7 @@ import (
 	"runtime"
 
-	"code.google.com/p/goauth2/oauth"
+	"google.golang.org/p/oauth2/v2"
```
Same thing here? `golang.org/x/oauth2`

odeke-em commented on go/README:
```diff
@@ -1,15 +1,15 @@
 
-   go get code.google.com/p/goauth2/oauth
-   go get code.google.com/p/google-api-go-client/youtube/v3
+   go get golang.org/x/oauth2/
```
Minor nit: replace oauth2/ with oauth2, so:
```
go get golang.org/x/oauth2
```

naoki-k commented:
@odeke-em I'm sorry for the delay in my reply. I fixed two points which you commented. Please review again?

naoki-k commented:
@odeke-em cc: @broady 
Thank you for your comments and I'm sorry for the delay in my reply.
I fixed some codes which you have commented and changed `oauth.go`'s implementation associated with changing from `code.google.com/p/goauth2/oauth` to `golang.org/x/oauth2`.
Then, I referenced #29 to implement the above.
Please review again.
